migration/sql-files/032-add-space-id-wilt.sql

```diff
@@ -1,5 +1,11 @@
 -- Alter the table work_item_link_types
-ALTER TABLE work_item_link_types ADD space_id uuid;
+INSERT INTO spaces (created_at, updated_at, id, name, description) VALUES (now(),now(),'2e0698d8-753e-4cef-bb7c-f027634824a2', 'system.space', 'Initial space');
```

Comment:
@aslakknutsen I wonder if this is a good place to start using migration done using only Go code + SQL instead of an SQL file. Then we don't have to duplicate the UUID at all. @hectorj2f do you know what I mean?

Something like this in `migration.go`:
```go
        // Version XY
	m = append(m, steps{
		func(tx *sql.Tx) error {
			return tx.Exec(fmt.Printf(`
-- Alter the table work_item_link_types
INSERT INTO spaces (created_at, updated_at, id, name, description) VALUES (now(),now(),'%[1]s', 'system.space', 'Initial space');
ALTER TABLE work_item_link_types ADD space_id uuid NOT NULL;
-- Add a default space_id to those tuples without one
ALTER TABLE work_item_link_types ALTER COLUMN space_id SET DEFAULT '%[1]s';
-- Once we set the values to the default. We drop this default constraint
ALTER TABLE work_item_link_types ALTER space_id DROP DEFAULT;

ALTER TABLE work_item_link_types ADD FOREIGN KEY (space_id) REFERENCES spaces(id) ON DELETE CASCADE;

-- Create indexes
CREATE INDEX ix_space_id ON work_item_link_types USING btree (space_id);			
`), space.SystemSpace.String())
		}
	})
```